app/charts/map/map.tsx

```diff
@@ -111,6 +106,9 @@ export const MapComponent = () => {
     showBaseLayer,
     showLabels: !areaLayer.show,
   });
+
+  const onHover = useOnHover();
```

Comment:
Not sure about this particular refactor. Here I have to dig into another file to understand that the tooltip is managed there. I am not even sure that the dispatch interaction and management of the tooltip are related so I think it's a bit awkard to have them tied together.